.eslintrc.json

```diff
@@ -53,7 +53,8 @@
         "@typescript-eslint/consistent-type-definitions": ["error", "interface"],
         "@typescript-eslint/consistent-type-imports": ["error"],
         "@typescript-eslint/method-signature-style": ["error", "method"],
-        "@typescript-eslint/sort-type-union-intersection-members": ["error"],
+        "@typescript-eslint/no-duplicate-enum-values": ["off"],
```

Comment:
Is there a reason to disable this rule?

Reply:
Yes: https://github.com/LSS-Manager/LSSM-V.4/actions/runs/5515490091/job/14932151280#step:15:835

![Screenshot_20230820-120913_Firefox Nightly.png](https://github.com/LSS-Manager/LSSM-V.4/assets/27238105/d632b9fb-3f6c-4ecd-b589-457f81f5d6d5)

due to duplicate status colors :)